### PR TITLE
Wasm dpp data contract create transition

### PR DESCRIPTION
**Pull request by fominok**
## Issue being fixed or feature implemented

Bindings for data contract create transition

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by qrayven — APPROVED** [review]

File: packages/wasm-dpp/test/unit/dataContract/stateTransition/DataContractCreateTransition/validation/state/validateDataContractCreateTransitionStateFactory.spec.js [hunk 1]
```diff
@@ -31,55 +35,54 @@ describe('validateDataContractCreateTransitionStateFactory', () => {
...
   });
 
-  it('should return invalid result if Data Contract with specified contractId is already exist', async () => {
-    stateRepositoryMock.fetchDataContract.resolves(dataContract);
+  // TODO wait for error types
+  // it('should return invalid result if Data Contract is already exist', async () => {
+  //   stateRepositoryMock.fetchDataContract.resolves(dataContract);
 
-    const result = await validateDataContractCreateTransitionState(stateTransition);
+  //   const result = await validateDataContractCreateTransitionState(stateTransition);
```
Comment (antouhou) [inline packages/wasm-dpp/test/unit/dataContract/stateTransition/DataContractCreateTransition/validation/state/validateDataContractCreateTransitionStateFactory.spec.js:53]: Okay, so what's the plan here? Consensus errors are now in the v24 branch, so technically it's possible to check this. Do you want to do it here or in a separate PR?

**Review by antouhou — APPROVED** [review]
Noice! It's so cool that you've made the StateRepository work. Good job!
